docs/user/quickstart.md

```diff
@@ -0,0 +1,240 @@
+* `slot` - some type of devices in the DC can occupy single position (U). An example could be blade systems, which could store blade servers inside(on the same U position). In this case we use 'slots' field to view them on the DC Visualization properly. You can set number of slots in previosly used 'Model' adding form, using 'layout of front/back size' fields.
+
+In our case, leave `slot` unset, and since we want to put our Blade System to the 6-position, type 6 in the `position` field.
+
+### Wrapping it up
+The last thing required is to fill barcode (for example: 123456) and save it.
+
+That's it.
+Congratulations!
+
+You've:
+* added new service, to use it everywhere
+* set-up new data center and added Rack
+* installed new asset
+
+That was easy! Now go to DataCenter -> Hardware to assets listing. And after that go to DC Vizualization to see the Rack on the map.
+
+## Data Center visualization
+
+### Finding your rack
+In this tutorial you will learn how to manage graphical representation of your data center.
+
+Let's go to the "Data Center visualization" menu item, and find your data center. You should see a new rack visible on the map.
+
+![quickstart-dcview](/img/quickstart-dcview.png "Data center view")
+
+Click on 'Edit' function and try to
+
+* drag rack to the new location
+* rotate it by clicking 'rotate' button
+* rename it by clicking on the pencil button
+
+Tips:
+
+> You can extend the data center layout by changing "Data center" grid columns and rows count.
+
+### Adding new racks directly from dc view
+
+You can quickly add multiple new racks from dc visualization.
+
+To do it, enter "Edit mode" and click on the view using "plus cursor" to add multiple racks.
+You can edit each of them in separation, but remember to click "Save" button when you're done.
+
+![quickstart-multiple-racks](/img/quickstart-multiple-racks.png "Multiple racks at once view")
+
+### Accesing the DC inspector
+
+Let's search for "Rack1" added at the beginning of tutorial. Click on dc rack view to get into the Rack detail view.
+
+<video width="500" height="500" controls>
+    <source src="/img/ralph-dc-inspecting.mov" type="video/mp4">
+    Your browser does not support video tag.
+</video>
+
+Both back and front  side of rack are displayed.
+
+### Accessing the edit asset form
+
+If you've played a bit with your beautiful rack, let's go back to the Asset View by clicking on the "Edit asset" in the Rack Inspector.
+
+Here we can explain the rest of non-obvious fields of the Asset.
+
+* `inventory number` -  could be your own, internal id, in many cases, such as: when you import some data from legacy systems.
+* `task url` - could be used in your existing workflow system such as jira, if you use it.
+* `force deprecation` - in some exceptions, you want to force given asset to be deprecated, even if it is still during the deprecation period.
+* TODO: `required support`
```

Comment:
`required support` - check if you know that this asset will require vendor support in the future.